test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -551,5 +551,22 @@ public async Task AddProductDefaultPKAndDifferentColumnOrderTest(SupportedLangua
             await this.SendOutputGetRequest("addproductdefaultpkanddifferentcolumnorder", null, TestUtils.GetPort(lang, true));
             Assert.Equal(1, this.ExecuteScalar("SELECT COUNT(*) FROM dbo.ProductsWithDefaultPK"));
         }
+
+        /// <summary>
+        /// Tests that when using a table with column names having slash (/ or \) in them 
+        /// we can insert data using output binding.
+        /// Excluding C#, JAVA and CSX languages since / or \ are reserved characters and are discouraged to use in variable names
```

Comment:
nit: It's not just discouraged, it's specifically not allowed. Would suggest making that clear to avoid confusion